modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/web/internal/layout/display/page/test/JournalArticleLayoutDisplayPageProviderTest.java

```diff
@@ -69,6 +75,48 @@ public void tearDown() throws Exception {
 		ServiceContextThreadLocal.popServiceContext();
 	}
 
+	@Test
+	public void testGetLayoutDisplayPageObjectProviderJournalArticleWithExpiredArticleVersionInfoItemReference()
+		throws Exception {
+
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext(_group.getGroupId());
+
+		JournalArticle originalArticle = JournalTestUtil.addArticle(
```

Comment:
@bakayattila, since we only use one journalArticle, we can avoid naming it "original*"

Reply:
The original in the case is meant to signify that this is the original version.